stdlib/2/sqlite3/dbapi2.pyi

```diff
@@ -120,6 +121,10 @@ class Connection(object):
     # set_progress_handler(handler, n) -> see https://docs.python.org/2/library/sqlite3.html#sqlite3.Connection.set_progress_handler
     def set_progress_handler(self, *args, **kwargs) -> None: ...
     def set_trace_callback(self, *args, **kwargs): ...
+    if sys.version_info >= (3, 7):
```

Comment:
I made the same change in 2 and 3 for this file to make sure the files stay mostly the same. I'll submit a followup PR to merge the two stubs.